tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = pep8,py36-inm{master,iso4}
+skip_missing_interpreters=True
+requires =
+    pip >= 21.0.1
+    setuptools
+
+[testenv:py36]
+basepython=python3.6
+
+[testenv]
+deps=
+    -rrequirements.txt
+    inmmaster: git+https://github.com/inmanta/inmanta-core.git@master
```

Comment:
Why not from artifacts?

Reply:
I copied this from the vscode repository. I agree it's better to use the artifacts.